### PR TITLE
add a README file

### PR DESCRIPTION
**Pull request by grote**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Public Transport Enabler
+========================
+
+This is a Java library allowing you to get data from public transport providers.
+Look into [AbstractNetworkProvider.java](https://github.com/schildbach/public-transport-enabler/blob/master/enabler/src/de/schildbach/pte/AbstractNetworkProvider.java) for an overview of the API.
```
Comment (schildbach) [inline README.md]: The file to look at would be NetworkProvider.java. And btw it's documented (-:

File: README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+Copy the `secrets.properties.template` file to `secrets.properties` like so:
+
+    $ cp enabler/test/de/schildbach/pte/live/secrets.properties.template enabler/test/de/schildbach/pte/live/secrets.properties
+
+You need to request the secrets directly from the provider. For Navitia based providers, you can [request a secret here](http://www.navitia.io/register).
+
+How to run live tests?
+----------------------
+
+Make sure the test you want to run does not require a secret and if it does, see above for how to get one.
```
Comment (schildbach) [inline README.md]: I think the straightforward way to run single tests is to run them via your IDE. Both IntelliJ and Eclipse have excellent support for JUnit tests.

**Comment by schildbach** [comment]
Thanks for taking the initiative on documentation!

File: README.md (hunk 2 above)
Comment (grote) [inline README.md]: I feel it is a lot harder to explain how to run these tests in these IDEs than just giving a command people can run. But feel free to add these explanations and use the command only as a fallback for people not using IDEs.

File: README.md (hunk 1 above)
Comment (grote) [inline README.md]: I see, please feel free to change the filename here.

**Comment by grote** [comment]
I updated this PR with the changes requested.

**Comment by schildbach** [comment]
Thanks! I have merged this, but moved it into enabler/ and moved the note about commenting out the test exclude a few lines down to the "command line" instructions.

Also I pushed a commit on top for a top-level README.md.

**Comment by grote** [comment]
Great, thank you!

**Pull request closed without merging**